open_academy/wizards/wizard_session.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class WizardSession(models.TransientModel):
+    _name = 'wizard_session'
+    _description = 'Wizard for session model'
+
+    session_ids = fields.Many2many('session', default=lambda self: self._context.get('active_ids'))
+    attendees_ids = fields.Many2many('res.partner')
+
+    def add_attendees(self):
+        for session in self.session_ids:
+            session.session_ids.attendees_ids |= session
```

Comment:
I don't know if I'm making the wrong suggestions, but let me explain

`self` is your model `wizard_session`, inside it, you have `session_ids`

If you do this `for session in self.session_ids` you are already inside the model `session`

I don't know if you tested the changes, but it will break, since session does not have `session_ids` (at least not that I remember)

```suggestion
        for session in self.session_ids:
            session.attendees_ids |= session
```

Reply:
I tested the changes with pdb, and I notice that session does not have the attribute **session_ids** but I forget to test all the workflow, sorry this will never happen again, thanks for the feedback.
Also I noticed that in this way the method will break, so I change session for self.attendees_ids

Reply:
Looks OK to me